MyRecipes/Views/Home/HomeView.swift

```diff
@@ -13,10 +13,19 @@ struct HomeView: View {
     
     var body: some View {
         NavigationView {
-            ScrollView {
-                RecipeList(recipes: recipesVM.recipes)
+            VStack {
+                if recipesVM.filteredRecipes.isEmpty {
+                    ContentUnavailableView.search
+                } else {
+                    ScrollView {
+                        RecipeList(recipes: recipesVM.filteredRecipes)
+                    }
+                }
             }
-            .navigationTitle("My Recipes")
+            .searchable(text: $recipesVM.searchText, prompt: "Find a recipe")
+            .navigationViewStyle(.stack)
```

Comment:
Since your is iOS 16 or higher, you can use NavigationStack instead of NavigationView, and that means you no longer need  .navigationViewStyle(.stack) modifier